doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,550 @@
+.. _ncs_release_notes_120:
+
+|NCS| v1.2.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.2.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for developing low-power wireless applications with nRF52 and nRF53 Series devices, though support for these devices is incomplete and not recommended for production.
+
+
+Highlights
+**********
+
```

Comment:
New highlight section (replacing what's there)
-	Added support for the new nRF5340 device 
-	New Multi-Protocol Service Layer (MPSL) included in nrfxlib (link?)
-	NFC supporting the new TNEP protocol  
-	New nRF9160 samples and libraries
-	General updates and bugfixes